test/villagebook/organisation/db_test.clj

```diff
@@ -5,6 +5,6 @@
 
 (deftest create-organisation
   (testing "Should create an organisation"
-    (let [{:keys [id]} (sut/create {:name "Nilenso" :color "pink"})]
+    (let [{:keys [id]} (sut/create stub/organisation)]
```

Comment:
Prefer factories instead, they are more versatile. You can hand roll them or use an existing library such as https://github.com/duck1123/clj-factory.